scan-engine/src/main/java/de/tillhub/scanengine/zebra/ZebraBarcodeScanner.kt

```diff
@@ -89,17 +96,43 @@ internal class ZebraBarcodeScanner(
         context.startActivity(intent)
     }
 
-    override fun dcssdkEventCommunicationSessionEstablished(scanner: DCSScannerInfo?) {
-        if (scanner == null) return
-        sdkHandler.dcssdkEnableAutomaticSessionReestablishment(true, scanner.scannerID)
+    override fun observeScanners(): Flow<List<Scanner>> = availableScannersFlow
+
+    override suspend fun connect(scannerId: String): ScannerResponse {
+        val result = sdkHandler.dcssdkEstablishCommunicationSession(scannerId.toInt())
```

Comment:
Conversion to Int has to be wrapped in try/catch. In case of exception returns Error.NotFound